src/transactions/simulation/TxSimUtils.cpp

```diff
@@ -212,10 +213,51 @@ mutateScaledAccountID(MuxedAccount& acc, uint32_t partition)
     return key;
 }
 
+PoolID
+getPoolID(LiquidityPoolConstantProductParameters const& params)
```

Comment:
Would it be possible to reuse logic we already have in the codebase to generate this id? i.e. OfferExchange::getPoolID

Reply:
I reworked some of this and think it makes sense to keep the new poolID methods separate from the one in OfferExchange.